test/e2e/manifest/pod-failover.json

```diff
@@ -30,13 +30,13 @@
         "masterProfile": {
             "count": 1,
             "dnsPrefix": "{dnsPrefix}",
-            "vmSize": "Standard_D4s_v3"
+            "vmSize": "Standard_D8s_v3"
         },
         "agentPoolProfiles": [
             {
                 "name": "agentpool1",
                 "count": 3,
-                "vmSize": "Standard_D4s_v3",
+                "vmSize": "Standard_D8s_v3",
```

Comment:
Why is this change made?

Reply:
Nvm, I just read the description.

Reply:
this is made so I can support up to 16 disks on a node. the pipelines run 1pod1pvc 1pod3pvc 3pod3pvc concurrently, so there are a maximum of 14 disk being attached to a node. doing this to avoid mount failures